dockers/docker-macsec/cli/show/plugins/show_macsec.py

```diff
@@ -220,21 +282,34 @@ def __init__(self, namespace_option, display_option):
             display_option, namespace_option)
 
     @multi_asic_util.run_on_multi_asic
-    def show(self, interface_name, dump_file):
+    def show(self, interface_name, dump_file, profile):
         global DB_CONNECTOR
         global COUNTER_TABLE
+        global macsec_profiles
```

Comment:
why is the variable, `macsec_profiles`, a global variable?

Reply:
made it a class variable instead of global.